proposals/simd/SIMD.md

```diff
@@ -294,7 +294,7 @@ return. The indices `i` in range `[0, 15]` select the `i`-th element of `a`. The
 indices in range `[16, 31]` select the `i - 16`-th element of `b`.
```

Comment:
This should IMO also mention what happens with out-of-range indices (e.g. validation error) for consistency with the variable index shuffle.

Reply:
My understand was that LaneIdx32 implies that indices must be 0-31. The encoding for these isn't documented atm.

Reply:
It is not strictly necessary to document these here. The definition of `LaneIdx32` states:

> [...] Many have a limited valid range, and it is a validation error if the immediate operands are out of range.
>
> ImmLaneIdx32: A byte with values in the range 0–31 identifying a lane.

but I think a "(note: [out-of-range indices are a validation error](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md#immediate-operands))" might be helpful.

Reply:
This is not the final spec text, so as long as the intended semantics and opcodes are clear it should be good to go.